src/main/java/org/apache/sling/feature/cpconverter/features/DefaultFeaturesManager.java

```diff
@@ -225,6 +224,45 @@ public void addConfiguration(String runMode, String pid, Dictionary<String, Obje
         }
     }
 
+    private void addAPIRegions(Feature feature, List<String> exportedPackages) {
```

Comment:
The regions java api supports generating (and parsing) of the extension JSON - we should not do this by hand anymore

Reply:
:) this class was already doing it like that - but I'll be happy to improve it to use the API.

Reply:
Thanks, yes, I know but I think if we touch the code we should improve :)

Reply:
You are right - this is better. I've just made the change. Thanks for reviewing @cziegeler !